pkg/grpc/default_api/api_pubsub.go

```diff
@@ -39,10 +40,20 @@ import (
 
 // Publishes events to the specific topic.
 func (a *api) PublishEvent(ctx context.Context, in *runtimev1pb.PublishEventRequest) (*emptypb.Empty, error) {
-	result, err := a.doPublishEvent(ctx, in.PubsubName, in.Topic, in.Data, in.DataContentType, in.Metadata)
```

Comment:
We can delete the `doPublishEvent` method and any other unused code in this package, if they won't be used any more.